pkg/server/serverpb/status.proto

```diff
@@ -1484,11 +1484,11 @@ message HotRangesResponseV2 {
     double qps = 3 [
       (gogoproto.customname) = "QPS"
     ];
-    // table_name indicates the SQL table that the range belongs to.
-    string table_name = 4;
-    // database_name indicates on database that has current hot range.
+    // table_name has been deprecated in favor of tables = 16;
+    string table_name = 4 [deprecated = true];
+    // database_name has been deprecated in favor of databases = 17;
     string database_name = 5;
-    // index_name indicates the index name for current range.
+    // index_name has been deprecated in favor of indexes = 17;
```

Comment:
is there a reason that only one field has `deprecated = true`?

Reply:
No this was an accident. I'll make the corresponding change to master.